### PR TITLE
fix: RSA removed, according to #164

### PR DESCRIPTION
**Pull request by peppelinux**
signing RSA algs removed, since we MUST support EC, according to https://github.com/italia/eudi-wallet-it-docs/issues/164

### CODE REVIEW

**Review by fmarino-ipzs — APPROVED** [review]

File: docs/en/relying-party-solution.rst [hunk 1]
```diff
@@ -11,6 +11,731 @@ This section describes how a remote Relying Party or a Verifier App requests to
 
 In this section the following flows are described:
 
+<<<
```
Comment (peppelinux) [inline docs/en/relying-party-solution.rst]: @asharif1990 this is a regression, please fix

**Comment by fventola-ipzs** [comment]
I suggest to add, as a note, the support of this alghoritms from Android 6+ and iOS 13.0+, like mentioned in the following official pages, that we can add also as bibliography, in particular for Android info [here](https://source.android.com/docs/security/features/keystore/features) and for iOS [here](https://developer.apple.com/documentation/cryptokit).

**Comment by Sh-Amir** [comment]
> I suggest to add, as a note, the support of this alghoritms from Android 6+ and iOS 13.0+, like mentioned in the following official pages, that we can add also as bibliography, in particular for Android info [here](https://source.android.com/docs/security/features/keystore/features) and for iOS [here](https://developer.apple.com/documentation/cryptokit).

So from the refrence that you provided it seems that on the mobile devices, there is no support for the brainpool curve.

File: docs/en/relying-party-solution.rst [hunk 2]
```diff
@@ -526,178 +527,38 @@ Below is a non-normative response example:
...
                     "keys": [
                         {
+                            "kty": "RSA",
                             "kty": "EC",
                             "use": "sig",
+                            "n": "1Ta-sE …",
+                            "e": "AQAB",
                             "crv": "P-256",
                             "x": "1kNR9Ar3MzMokYTY8BRvRIue85NIXrYX4XD3K4JW7vI",
                             "y": "slT14644zbYXYF-xmw7aPdlbMuw3T1URwI4nafMtKrY",
```
Comment (peppelinux) [inline docs/en/relying-party-solution.rst]: y and x not here @asharif1990

File: docs/en/relying-party-solution.rst [hunk 3]
```diff
@@ -505,19 +507,18 @@ Below is a non-normative response example:
 .. code-block:: text
 
     {
+        "alg": "RS256",
```
Comment (peppelinux) [inline docs/en/relying-party-solution.rst]: RS256 not here @asharif1990

File: docs/en/relying-party-solution.rst [hunk 4]
```diff
@@ -526,178 +527,38 @@ Below is a non-normative response example:
                     "kid": "2HnoFS3YnC9tjiCaivhWLVUJ3AxwGGz_98uRFaqMEEs"
                 }
             ]
-        },
-        "metadata": {
-            "wallet_relying_party": {
-                "application_type": "web",
-                "client_id": "https://rp.example.it",
-                "client_name": "Name of an example organization",
```
Comment (peppelinux) [inline docs/en/relying-party-solution.rst]: this is required @asharif1990

**Comment by fventola-ipzs** [comment]
> > I suggest to add, as a note, the support of this alghoritms from Android 6+ and iOS 13.0+, like mentioned in the following official pages, that we can add also as bibliography, in particular for Android info [here](https://source.android.com/docs/security/features/keystore/features) and for iOS [here](https://developer.apple.com/documentation/cryptokit).
> 
> So from the refrence that you provided it seems that on the mobile devices, there is no support for the brainpool curve.

You're right, we need external library to manage brainpool curve on mobile. In Android are implemented with [BouncyCastle](https://www.bouncycastle.org/) and in iOS with [Swift extension wrapper](https://github.com/gematik/OpenSSL-Swift).

File: docs/en/pid-eaa-data-model.rst [hunk 5]
```diff
@@ -269,6 +269,16 @@ The corresponding SD-JWT verson for PID is given by
...
+    "cnf": {
+      "jwk": {
+          "crv": "P-256",
+          "kty": "EC",
+          "x": "qrJrj3Af_B57sbOIRrcBM7br7wOc8ynj7lHFPTeffUk",
+          "y": "1H0cWDyGgvU8w-kPKU_xycOCUNT2o0bwslIQtnPU6iM",
+          "kid": "5t5YYpBhN-EgIEEI5iUzr6r0MR02LnVQ0OmekmNKcjY"
+        }
+    },
+    "type": "PersonIdentificationData",
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]: @asharif1990 

type is not allowed here, we only use `vct`

File: docs/en/pid-eaa-data-model.rst [hunk 6]
```diff
@@ -491,6 +501,16 @@ The corresponding SD-JWT for the previous data is represented as follow, as deco
...
+      "jwk": {
+        "crv": "P-256",
+        "kty": "EC",
+        "x": "qrJrj3Af_B57sbOIRrcBM7br7wOc8ynj7lHFPTeffUk",
+        "y": "1H0cWDyGgvU8w-kPKU_xycOCUNT2o0bwslIQtnPU6iM",
+        "kid": "5t5YYpBhN-EgIEEI5iUzr6r0MR02LnVQ0OmekmNKcjY"
+      }
+    },
+    "type": "HealthInsuranceData",
     "vct": "DisabilityCard",
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
    "vct": "DisabilityCard",
```

File: docs/en/pid-eaa-issuance.rst [hunk 7]
```diff
@@ -982,6 +991,9 @@ Below is a non-normative example of an Entity Configuration containing an `openi
               {
                 "format": "vc+sd-jwt",
                 "cryptographic_binding_methods_supported": ["jwk"],
+
+                "cryptographic_suites_supported": ["ES256", "ES512"],
+
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst:996]:
```suggestion
                "cryptographic_suites_supported": ["ES256", "ES512"],
```

Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
                "cryptographic_suites_supported": ["ES256", "ES512"],
```

Deprecated in Draft 13, it must be `credential_signing_alg_values_supported`

File: docs/en/pid-eaa-issuance.rst [hunk 8]
```diff
@@ -990,6 +1002,7 @@ Below is a non-normative example of an Entity Configuration containing an `openi
                       ]
                   }
                 },
+
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
```

File: docs/en/relying-party-solution.rst [hunk 9]
```diff
@@ -1,21 +1,742 @@
```
Comment (peppelinux) [inline docs/en/relying-party-solution.rst]: mmhh no,

this was moved in the file remote-flow.rst, therefore it must not be copied in this file

File: docs/en/algorithms.rst [hunk 10]
```diff
@@ -8,91 +8,123 @@ Cryptographic algorithms
...
   :header-rows: 1
 
-  * - **Algorithm**
+  * - **Algorithm `alg` parameter value**
+    - **Description**
     - **Operations**
     - **References**
-  * - **RS256** 
+  * - **ES256** 
+    - Elliptic Curve Digital Signature Algorithm (ECDSA) using one of the Agreed curves and SHA256.
```
Comment (peppelinux) [inline docs/en/algorithms.rst]:
```suggestion
    - Elliptic Curve Digital Signature Algorithm (ECDSA) using one of the enabled curves listed in the section below and SHA256.
```

File: docs/en/algorithms.rst [hunk 11]
```diff
@@ -8,91 +8,123 @@ Cryptographic algorithms
...
     - Signature
-    - :rfc:`7518`.
-  * - **RS512**
+    - :rfc:`7518`, `[SOG-IS] <https://www.sogis.eu/documents/cc/crypto/SOGIS-Agreed-Cryptographic-Mechanisms-1.3.pdf>`_, `[ETSI] <https://www.etsi.org/deliver/etsi_ts/119300_119399/119312/01.04.03_60/ts_119312v010403p.pdf>`_ .
+  * - **ES384** 
+    - Elliptic Curve Digital Signature Algorithm (ECDSA) using one of the Agreed curves and SHA384.
```
Comment (peppelinux) [inline docs/en/algorithms.rst]:
```suggestion
    - Elliptic Curve Digital Signature Algorithm (ECDSA) using one of the enabled curves listed in the section below and SHA384.
```

File: docs/en/algorithms.rst [hunk 12]
```diff
@@ -8,91 +8,123 @@ Cryptographic algorithms
...
     - Signature
-    - :rfc:`7518`.
-  * - **RSA-OAEP**
-    - Key Encryption
-    - :rfc:`7518`.
+    - :rfc:`7518`, `[SOG-IS] <https://www.sogis.eu/documents/cc/crypto/SOGIS-Agreed-Cryptographic-Mechanisms-1.3.pdf>`_, `[ETSI] <https://www.etsi.org/deliver/etsi_ts/119300_119399/119312/01.04.03_60/ts_119312v010403p.pdf>`_ .
+  * - **ES512**
+    - Elliptic Curve Digital Signature Algorithm (ECDSA) using one of the Agreed curves and SHA521.
```
Comment (peppelinux) [inline docs/en/algorithms.rst]:
```suggestion
    - Elliptic Curve Digital Signature Algorithm (ECDSA) using one of the enabled curves listed in the section below and SHA521.
```

**Review by fmarino-ipzs — APPROVED** [review]

**Pull request merged**